modules/RTC/TraceablePeerConnection.js

```diff
@@ -1495,6 +1543,42 @@ TraceablePeerConnection.prototype.setAudioTransferActive = function(active) {
     return changed;
 };
 
+/**
+ * Takes in a *unified plan* offer and inserts the appropriate
+ * parameters for adding simulcast receive support.
+ * @param {} A session description object (with 'type' and 'sdp' fields)
+ * @return {} A session description object with its sdp field modified to
+ * advertise simulcast receive support
+ */
+TraceablePeerConnection.prototype._insertUnifiedPlanSimulcastReceive
+    = function(desc) {
+        const sdp = transform.parse(desc.sdp);
+        const video = sdp.media.find(mline => mline.type === 'video');
+
+        // In order of lowest to highest spatial quality
+        video.rids = [
+            {
+                id: SIM_LAYER_1_RID,
+                direction: 'recv'
+            },
+            {
+                id: SIM_LAYER_2_RID,
+                direction: 'recv'
+            },
+            {
+                id: SIM_LAYER_3_RID,
+                direction: 'recv'
+            }
+        ];
+        // eslint-disable-next-line camelcase
+        video.simulcast_04 = {
```

Comment:
yeah this is weird...it's a side-effect of the implementation of sdp-transform.  there are 2 implementations of the unified plan spec, firefox does the "03" draft, so this field was named `simulcast_03` (there's a typo here) to show that it reflects the 03 draft